RunesQuery.lua

```diff
@@ -270,7 +66,8 @@ function h.getQuery(args)
 			'COALESCE(Teams.Short,Teams.Name,ST.Team) = Team',
 			'SP.Champion',
 			'SP.IngameRole = Position',
-			-- 'SP.KeystoneMastery=Keystone', -- unused ??
+			'SP.PrimaryTree = Primary',
```

Comment:
can we instead standardly call them PrimaryTree & SecondaryTree (so update INTRO_COLUMNS and the i18n and don't alias here)

otherwise looks good 👍

Reply:
Yay standards :D
Fixing~